### ISSUE TITLE
How to disable AwasomeTTS error messages?

### ISSUE THREAD

**Issue by eugen-nesterov** [AwesomeTTS/awesometts-anki-addon#260]
In cases when I have no internet connection I constantly receive these annoying notifications from AwasomeTTS while studying new cards. How do I get rid of them?

<img width="612" alt="Screen Shot 2022-04-18 at 15 29 45" src="https://user-images.githubusercontent.com/87163666/164029553-eb7fb66f-86eb-445c-8e93-417ce98b7d38.png">
<img width="864" alt="Screen Shot 2022-04-19 at 13 18 13" src="https://user-images.githubusercontent.com/87163666/164029567-c32cd3c0-ef3a-4930-b2ab-59f725740e5a.png">
<img width="864" alt="Screen Shot 2022-04-19 at 17 21 19" src="https://user-images.githubusercontent.com/87163666/164029575-14f730d4-788b-476b-90e8-9b9b5cfba54f.png">
<img width="864" alt="Screen Shot 2022-04-19 at 17 24 03" src="https://user-images.githubusercontent.com/87163666/164029577-c3f2d0d5-8e63-4eb8-9f5e-d6d0530133cf.png">
<img width="864" alt="Screen Shot 2022-04-19 at 17 24 14" src="https://user-images.githubusercontent.com/87163666/164029584-a3ad957b-fe22-4d95-9008-4b25e37764bc.png">
<img width="864" alt="Screen Shot 2022-04-19 at 17 26 16" src="https://user-images.githubusercontent.com/87163666/164029591-a9206c02-c2ea-4b43-9d85-e1bae8bd42c3.png">
<img width="864" alt="Screen Shot 2022-04-19 at 17 26 27" src="https://user-images.githubusercontent.com/87163666/164029596-44420ee1-55ba-467c-8cf6-2962ed5e5c24.png">

**Comment by luc-languagetools** [comment]
Do you need the sound at all ? Can you remove the TTS tag ?

**Comment by eugen-nesterov** [comment]
@luc-languagetools Of course, I need it! It's extremely helpful while learning foreign languages (English in my case). How can you learn a foreign language without pronunciation?

As far as I know, previously there was a "Playback" tab with a "Show errors" checkmark in the AwasomeTTS settings to disable/enable these messages:
![2022-04-20 04 26 57](https://user-images.githubusercontent.com/87163666/164128811-a97a6c1e-d373-4437-8707-bee10aef6aca.png)
But at present, it's gone. How to bring it back? Is there any other alternative way to disable these error messages?

**Comment by EugeneTarasenko** [comment]
Still no way to fix this issue?

**Comment by luc-languagetools** [comment]
Remove the TTS tag, and add it back by following this tutorial: https://languagetools.anki.study/tutorials/awesometts-on-the-fly-tts
if you want more reliable TTS audio, consider using batch generation: https://languagetools.anki.study/tutorials/awesometts-batch-generation
